src/nl/hannahsten/texifyidea/lang/commands/LatexDiffcoeffCommand.kt

```diff
@@ -0,0 +1,38 @@
+package nl.hannahsten.texifyidea.lang.commands
+
+import nl.hannahsten.texifyidea.lang.LatexPackage
+import nl.hannahsten.texifyidea.lang.LatexPackage.Companion.SIUNITX
+
+/**
+ * @author Hannah Schellekens
+ */
+enum class LatexSiunitxCommand(
```

Comment:
Fix name, author, usage

Reply:
I fixed name and author, but what should I fix with usage?

Reply:
Sorry I meant adding it in LatexMathCommand

Reply:
Consider it done 👍